server/clip_server/model/flash_attention.py

```diff
@@ -0,0 +1,130 @@
+import torch
```

Comment:
Is this module code implemented by us? If not, please provide the copyright info.

Reply:
By us